tests/utils.py

```diff
@@ -19,11 +19,11 @@
 
 def og_test(func):
     def wrapper():
-        assert_test_scene()
+        env = assert_test_env()
```

Comment:
assert_test_env already puts env in the global scope so no need for the = here. Instead, let's put the declaration of env even above this function instead of right before assert_test_env.